zowex/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "zowe"
-version = "1.0.2"
+version = "1.0.3"
```

Comment:
Dumd questions:
- Curious if we really need to publish a new version of the daemon when we are mostly reformatting the code (with rust-analyzer)?
  - Is the reason that this PR will remove the `clippy` warning that comes from `util_get_zowe_env`?
    - Does the end user see this warning?

Reply:
There is a bugfix in the Rust code in the first commit: https://github.com/zowe/zowe-cli/pull/1652/commits/7a17ce6c1d4f9288ff5da56e1b23c28dec192456
I believe that warrants the patch release 😛

Reply:
thanks for clarifying! 🥳